source/applications/advanced/gamma_correction.py

```diff
@@ -0,0 +1,100 @@
+"""
+Capture a 2D image and apply a configurable gamma correction.
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    """Configure argparse.
+
+    Returns:
+        argparser arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        description="Capture 2D image and apply gamma correction\
+            \nExample:\n\t $ python gamma_correction.py 2",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    parser.add_argument(
+        "gamma", nargs="?", type=float, help="Gamma correction value", default="2"
```

Comment:
Why have a default value? Is this the value most users would normally use? Would it make sense to make the user always input what correction value they want?

Reply:
@SatjaSivcev 
Again, I didn't want to modify the nature of the original example. I actually agree with you Trym, and I would maybe propose that we comment on the expected values instead of a default.

```python
parser.add_argument(
    "gamma", type=float, help="G < 1 makes image darker, while G > 1 makes image lighter"
)
```